internal/repository/employee/employee_test.go

```diff
@@ -0,0 +1,439 @@
+package employee
+
+import (
+	"context"
+	"fmt"
+	conf "github.com/dilyara4949/employees-api/internal/config"
+	"github.com/dilyara4949/employees-api/internal/database"
+	"github.com/dilyara4949/employees-api/internal/domain"
+	"github.com/dilyara4949/employees-api/internal/repository/position"
+	"log"
+	"reflect"
+	"testing"
+)
+
+func SetEnv(t *testing.T) {
+	cfg := map[string]string{
```

Comment:
I would suggest having these parameters stored in `.env` file (not in cvs) and read it when you running integration tests with build constraints.

Reply:
+